### PR TITLE
Change order, refrase some sentences, remove unnecessary information

### PR DESCRIPTION
**Pull request by lvarin**
About creation of Rahti projects

### CODE REVIEW

File: docs/cloud/rahti/usage/projects_and_quota.md [hunk 1]
```diff
@@ -53,73 +62,32 @@ click on "Save". After doing this, the default billing project
...
-Once you have filled in the fields, click "Create", and you will see the application
-catalog where you can pick an application template or import your
-own one.
-
-For more information about using the web interface, refer to the
-[official OpenShift documentation](https://docs.okd.io/). You can find
-out which version of the documentation to look at in the web interface by
-clicking the question mark symbol in the top bar and selecting "About".
+!!! note
+    It is possible to change the description of a project, but **it will not** change the csc_project label**.
```
Comment (JMuff22) [inline docs/cloud/rahti/usage/projects_and_quota.md:90]: There is some formatting error here. Which part should in in **bold text** ?

Comment (lvarin) [reply docs/cloud/rahti/usage/projects_and_quota.md:90]: Second bold was unnecessary. I added the word billing explicitly.\

thank you for the review

**Comment by JMuff22** [comment]
I removed the formatting error. @lvarin can you confirm the change? If so, this is ready to be merged :)

**Comment by lvarin** [comment]
Looks good to me now. Thanks for the fix

**Review by joonas-somero — APPROVED** [review]

**Pull request merged**